src/overlays/actors/ovl_En_Holl/z_en_holl.h

```diff
@@ -4,6 +4,33 @@
 #include "ultra64.h"
 #include "global.h"
 
+#define ENHOLL_GET_TYPE(thisx) (((thisx)->params >> 6) & 7)
+#define ENHOLL_GET_SWITCH_FLAG(thisx) ((thisx)->params & 0x3F)
+
+/**
+ * Two kinds of holls:
+ *   Horizontal (H):
+ *     For the player to move horizontally between rooms.
+ *     Cuboid-shaped, 200 units high plus 50 units into the ground, 400 units wide or, if narrow, 200 units wide.
+ *   Vertical (V):
+ *     For the player to move vertically between rooms.
+ *     Cylinder-shaped, radius of 120 units.
+ *
+ * Three kinds of visual effects:
+ *   Visible: A plane is drawn at the holl location, and fades away as the player approaches.
+ *   Fade: The whole screen fades black as the player approaches.
+ *   Invisible: Approaching the holl does not cause any particular visual effect.
```

Comment:
also, may be good to overall refer to it as BG_FADE?
when reading most of the code in this review, i read "fade" as the types that themselves fade, which you have called "visible" currently

Reply:
"bg_fade" would be unfortunate because it'd break my fantastic naming scheme of "one word = one feature",
for example `ENHOLL_V_DOWN_FADE_LARGE` = vertical + down + fade + large
and `ENHOLL_V_DOWN_BG_FADE_LARGE` = vertical + down + bg + fade + large ? <img src="https://cdn.discordapp.com/emojis/643089269614968851.webp?size=240&quality=lossless" alt="peepoCryTeddy" height="24">

Could be alleviated with "bgfade" instead (`ENHOLL_V_DOWN_BGFADE_LARGE`)
(though not in function named `EnHoll_VerticalDownBgFadeLarge` but that's less important)

With that one play state unk being `bgCoverAlpha` maybe `BGCOVER` would be better? I don't think you could mistake that for the "plane fading" (visible) kind of holl

Reply:
I pushed fade -> bgcover, lmk how it looks